resources/views/emails/student-verification.blade.php

```diff
@@ -76,16 +78,14 @@
                                     </div></td></tr></tbody></table></div><!--[if mso | IE]>
                     </td><td style="vertical-align:top;width:420px;">
                     <![endif]--><div aria-labelledby="mj-column-per-70" class="mj-column-per-70" style="vertical-align:top;display:inline-block;font-size:13px;text-align:left;width:100%;"><table cellpadding="0" cellspacing="0" width="100%" border="0"><tbody><tr><td style="word-break:break-word;font-size:0px;padding:0px 0px 10px;" align="left"><div style="cursor:auto;color:#222228;font-family:'Avenir Next', Avenir, sans-serif;font-size:16px;line-height:30px;">
```

Comment:
@minhquang97 sao lai toàn style inline thế này e ?

Reply:
2 cái email xác thực này em dùng template a.